airbyte-webapp/src/views/Connector/ServiceForm/components/Sections/auth/AuthButton.module.scss

```diff
@@ -0,0 +1,23 @@
+@use "../../../../../../scss/colors";
+@use "../../../../../../scss/variables";
+
+.authSectionRow {
+  display: flex;
+  align-items: center;
+}
+
+.message {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 17px;
+  margin-left: variables.$spacing-lg;
+}
```

Comment:
Given that https://github.com/airbytehq/airbyte/issues/14848 is now closed I think it would be nice if we could still move this over to use the new text components instead, so we're not introducing new custom font-styles here.